src/app/core/http/api-prefix.interceptor.ts

```diff
@@ -18,7 +18,14 @@ export class ApiPrefixInterceptor implements HttpInterceptor {
    * Intercepts a Http request and prefixes it with `environment.serverUrl`.
    */
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    request = request.clone({ url: environment.serverUrl + request.url });
+
+    /**
+     * Ignore URLs that are complete for i18n
+     */
+    if (!request.url.includes('http:') && !request.url.includes('https:')) {
```

Comment:
Would you mind elaborating the logic of this if condition?

Reply:
This solution is part of another PR that I also sent to correct the i18n